src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -44,4 +47,4 @@ add_onnx_mlir_library(NNPAAccel
   OMZHighOps
   OMZLowOps
   )
-
+add_dependencies(NNPAAccel zdnn)
```

Comment:
You can add DEPENDS in the `add_onnx_mlir_library` call instead.

Reply:
Perfect. Should have read MLIR.cmake more carefully. Thanks.